tests/playground.html

```diff
@@ -586,6 +591,7 @@
           </shadow>
         </value>
       </block>
+      <label text="Input/Output:" web-style="ioLabel"></label>
```

Comment:
I'll name it web-class, then.  iOS side wants it to be clear that setting this may not set the style in iOS, since they may be implementing it quite differently.